skopt/callbacks.py

```diff
@@ -176,3 +178,28 @@ def _criterion(self, result):
                 return False
         else:
             return None
+
+
+class DeltaYStopper(EarlyStopper):
+    """Stop the optimization if the `n_best` minima are within `delta`
+
+    Stop the optimizer if the absolute difference between the `n_best`
+    objective values is less than `delta`.
+    """
+    def __init__(self, delta, n_best=5):
```

Comment:
Not sure. This implements "delta between N best", with N=2 that would be "delta between best and last added". If we want "delta between last two points added" that would be something different.

My thinking was that because of exploration you might not want "delta between last N" but "delta between N best".